project2/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Anton|Cabin|Fjalla+One|Quicksand&display=swap" rel="stylesheet">
+    <title>destiny2gamecard</title>
+    <STYLE>
+        A {
+            text-decoration: none;
+        }
+    </STYLE>
+</head>
+
+
+<body>
+    <div class="card">
+        <div class="star">
+            <img src="images/starts.png" alt="4.5 star rating"></div>
+        <div class="rating">
+            <p>8.9/10</p>
+        </div>
+        <div class="fps">
+            <p>First-person shooter</p>
+        </div>
+        <img src="images/title.png" alt="text title of destiny 2 shadowkeeper">
```

Comment:
You can replicate just the text of your image in your `alt` here because you want a screenreader to have the same experience as a sighted user.
```suggestion
        <img src="images/title.png" alt="Destiny 2 Shadowkeeper">
```